commands.js

```diff
@@ -824,7 +824,7 @@ exports.commands = {
 			return this.errorReply("User '" + name + "' is unregistered, and so can't be promoted.");
 		}
 
-		let currentGroup = ((room.auth && room.auth[userid]) || (room.isPrivate !== true && Users.usergroups[userid]) || ' ').charAt(0);
+		let currentGroup = room.getAuth({userid, group: Users.usergroups[userid]}) || ' ';
```

Comment:
Is this (using `userid` instead of `userid: userid`) supported by Node 4.x?

Also, this needs to be `(Users.usergroups[userid] || ' ').charAt(0)` and not `Users.usergroups[userid]`